cmd/corald/routes/routes.go

```diff
@@ -65,6 +65,9 @@ func routes(a *app.App) {
 				return "/v1/exec/" + c.Params["query_set"]
 			}))
 
-	a.Handle("POST", "/v1/item", fixtures.Handler("items/itemid", http.StatusCreated))
-	a.Handle("PUT", "/v1/item", handlers.Proxy(spongedURL, nil))
+	a.Handle("PUT", "/v1/item",
```

Comment:
I propose we accept both PUT and POST for the upsert endpoint. This follows the clear web semantics around CRUD: POST for Create, PUT for Update. "Upsert" does both so we should accept both.